loops-arrays-objects.js

```diff
@@ -49,20 +64,64 @@ const scores = [
 //  ];
 // ==========================================
 
-
-
-
-
-// ==========================================
-// Opdracht 2
-// Schrijf een script die e-mailadressen genereert voor al onze medewerkers. Sla dit op in een nieuwe property "email" die je toevoegt aan iedere medewerker.
-
+console.log(scores)
+
+for (let i = 0; i < scores.length; i++) {
+    const score = scores[i].score;
+
+    switch (true) {
+        case score < 60:
+            console.log("F");
+            scores[i].grade = "F";
+            break;
+        case score < 70:
+            console.log("D");
+            scores[i].grade = "D";
+            break;
+        case score < 80:
+            console.log("C");
+            scores[i].grade = "C";
+            break;
+        case score < 90:
+            console.log("B");
+            scores[i].grade = "B";
+            break;
+        case score < 100:
+            console.log("A");
+            scores[i].grade = "A";
+    }
```

Comment:
```suggestion
    switch (true) {
        case score < 60:
            scores[i].grade = "F";
            break;
        case score < 70:
            scores[i].grade = "D";
            break;
        case score < 80:
            scores[i].grade = "C";
            break;
        case score < 90:
            scores[i].grade = "B";
            break;
        case score < 100:
            scores[i].grade = "A";
    }
    
    console.log(scores[i].grade);
```

Scheelt weer wat code